bin/bank/pycbc_brute_bank

```diff
@@ -0,0 +1,231 @@
+#!/usr/bin/env python
```

Comment:
Please add a top docstring with the purpose and use it as the description in the ArgumentParser.